docs/products/grafana/reference/plugins.rst

```diff
@@ -95,7 +95,7 @@ Status history
 Status Panel - `Grafana <https://grafana.com/grafana/plugins/vonage-status-panel/>`__ | `GitHub <https://github.com/Vonage/Grafana_Status_panel>`__
     Use as a centralized view for the status of component in a glance.
 
-Statusmap - `Grafana <https://grafana.com/grafana/plugins/flant-statusmap-panel/>`__ | `GitHub <https://github.com/flant/grafana-statusmap>`__
+``Statusmap`` - `Grafana <https://grafana.com/grafana/plugins/flant-statusmap-panel/>`__ | `GitHub <https://github.com/flant/grafana-statusmap>`__
```

Comment:
(I'll comment once although we should apply whatever we agree to a few instances of this). I think Statusmap is a proper noun in this context, not a command or keyword that needs the code/literal markup. But I guess you did this for a reason, so can we discuss? It's like we'll create additional documentation about how to do things with the Statusmap plugin in future, for example, so this might not always be the only usage.

Reply:
Added the three words like this to the dictionary, and made them "plain" in this file.